app/views/users/_post.html.erb

```diff
@@ -0,0 +1,11 @@
+<li>
+
+<h4>
+<%= link_to "Post##{post.id}", user_post_path(post.author, post.id) %>
```

Comment:
- [ ] Instead of rendering the Post id, let's render the Post title instead, like so

  ```
  <%= link_to post.title, user_post_path(post.author, post.id) %>
  ```

Reply:
Well resolved 👍🏽